deploy/lib/server_config.rb

```diff
@@ -112,7 +112,7 @@ def info
       logger.info "<info>"
       logger.info "\s\s<isJar>#{@@is_jar}</isJar>"
       logger.info "\s\s<properties>"
-      @properties.sort {|x,y| y <=> x}.each do |k, v|
+      @properties.sort {|x,y| x <=> y}.each do |k, v|
```

Comment:
If I remember right, the reason we put this sort in place is to make sure we replace the longest properties first. That way, if we see "@ml.app-name-foo-bar" and we have properties "app-name" and "app-name-foo", we want to use "app-name-foo" for the substitution. 

What's the reason for changing the sort order?